CMakeLists.txt

```diff
@@ -78,99 +154,91 @@ elseif(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
     endif()
 endif()
 
-set(SIMBODY_MAJOR_VERSION 3)
-set(SIMBODY_MINOR_VERSION 8)
-set(SIMBODY_PATCH_VERSION 0)
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED TRUE)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
```

Comment:
I be refactoring this slightly in an upcoming PR(s), because this is a CMake global setting (i.e. shouldn't set when a subproject). The proper method is to add C(XX)_STANDARD properties to the simbody library targets.